flang/test/Lower/OpenMP/omp-parallel-region.f90

```diff
@@ -0,0 +1,84 @@
+! This test checks lowering of OpenMP parallel Directive with arbitrary code
```

Comment:
Can you add more tests?
1) With a fir loop inside.
2) With an if statement.
3) with a goto statement (branching to inside the region).

Reply:
Sure will take care of this in next revision.